packs/github/actions/lib/base.py

```diff
@@ -8,19 +8,20 @@
     'BaseGithubAction'
 ]
 
-BASE_URL = 'https://github.com'
-
 
 class BaseGithubAction(Action):
     def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('BASE_URL', None)
+        base_url = base_url or None
```

Comment:
This line is redundant: `.get('BASE_URL', None)` means default to `None` if `base_url` is not set.